rasterio/rio/info.py

```diff
@@ -38,6 +128,13 @@ def edit(ctx, input, nodata, crs, transform, tags):
     or JSON-encoded GDAL geotransform arrays like
 
       [101985.0, 300.038, 0.0, 2826915.0, 0.0, -300.042]
+
+    Metadata items may also be read from an existing dataset using a
+    combination of the --like, --crs, --nodata, --transform, and --all
+    options.
+
+      rio-edit-info example.tif --all --like template.tif
```

Comment:
Please consider updating the docstring comments for ```--transform``` because the value must be string quoted but isn't demonstrated that way:

```
--transform '[300.038, 0.0, 101985.0, 0.0, -300.042, 2826915.0]'
```

I don't know that calling them JSON-encoded makes that super clear in this context; they are just plain arrays, in quotes...